dask_geopandas/geohash.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+import pandas as pd
+from pygeohash import encode
```

Comment:
I would rather not have `pygeohash` as a hard dependency. Can you make it optional and raise if it is not available?

Reply:
Updated in 5aca5f50. I have updated core.py (to not raise import errors) and geohash.py (to raise import errors if called). I think this works ok but I haven't dealt with optional dependencies before so there may be a better way how to handle this?